src/index.html

```diff
@@ -2,16 +2,182 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
+    <meta 
+    name="viewport"
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    
+    <form action="http://localhost:3000/api" method="post">
+      <fieldset>
+        <legend>Personal information</legend>
+        
+        <div class="form-field">
+          <label for="Surname">
+            Surname:
+          </label>
+      
+          <input type="text" name="Surname" 
+            id="Surname" required
+          >
```

Comment:
```suggestion
          <input
            type="text"
            name="Surname" 
            id="Surname"
            required
          >
```

![image](https://user-images.githubusercontent.com/80700247/226931659-af8af163-e311-4ccc-88d1-d847e3fc3a60.png)

Fix all  such cases